apps/admin-portal/src/components/side-panel.tsx

```diff
@@ -1,186 +0,0 @@
-/**
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import * as React from "react";
```

Comment:
Couldn't we separate the components into different files in order to make it more readable? WDYT?

Reply:
Actually this file was refactored and was moved to the new `@wso2is/react-components` module [1]. There, the components have been granularized. Please follow the bellow link, and let me know if you have any further change requests.

[1] https://github.com/wso2/identity-apps/tree/c1d5d905dc1b218ea6e07b00fe5b2b6c92719bc7/modules/react-components/src/side-panel

Reply:
Looks good.